apps/dev-proxy.config.js

```diff
@@ -1,15 +1,15 @@
-const HOST = process.env.MOLGENIS_APPS_HOST || "https://emx2.dev.molgenis.org";
```

Comment:
why does this give console errors ? , suggest to keep default pointing to dev server

Reply:
Agreed, this should not have been in the commit